[ADD] accounting: add self-billing instructions (#17336)
odoo/documentation: content/applications/finance/accounting/vendor_bills.rst

   Once confirmed, a vendor bill can no longer be updated. Click :guilabel:`Reset to draft` if
   changes are required.

.. _accounting/vendor_bills/self-billing:

Self-billing
============

Self-billing is when a customer generates a vendor bill on the supplier's behalf and sends a copy of

> the vendor bill to the supplier for their records. This workflow removes the need for the supplier

Reviewer: ```suggestion
Self-billing is a workflow where a customer generates a vendor bill on the supplier's behalf and sends a copy of to the supplier for their records. It removes the need for the supplier
```
- *self-billing is when* wounds a bit weird to me? 
- I think we can remove the repetition of *vendor bill*
- to avoid repeating workflow

if you have a better solution that prevents these 3 issues, feel free to use it!

to send the customer a vendor bill and is common in industries where the customer has more reliable

> transaction data. It also reduces the time to pay the supplier as the customer does not have to wait

Reviewer: ```suggestion
transaction data. It also reduces payment time as the customer does not have to wait
```

for the supplier to issue a vendor bill.

Odoo supports the generation of both self-billing vendor bills and self-billing credit notes.
Sending these self-billing vendor bills and credit notes is supported in Odoo both by email and via

> Peppol.

Reviewer: ```suggestion
Odoo supports the generation of both self-billing vendor bills and self-billing credit notes, which can be sent by email and via Peppol.
```
to avoid repetitions (self-billing vendor bills/credit notes + odoo + support)

.. _accounting/vendor_bills/self-billing/config:

Configuration
-------------

To configure self-billing in Odoo, first create a separate purchase journal specifically for
self-billing:

#. Navigate to :menuselection:`Accounting --> Configuration --> Journals`.
#. Click the :guilabel:`New` button.
#. Enter a :guilabel:`Journal Name` and set the :guilabel:`Type` to :guilabel:`Purchase`.
#. In the :guilabel:`Advanced Settings` tab, enable :guilabel:`Self Billing`.

.. note::
   The sequence of self-bills contains the :ref:`database ID <export_import_data/different-ids>`

>    (also known as the internal ID) of the partner, making the sequence unique per partner.

Reviewer: Since this isn't really related to the journals (as far as I understand?) or the configuration, should be maybe move this note to another section (*Create a self-bill* maybe?)

either create them directly from the :ref:`self-billing purchase journal
<accounting/vendor_bills/self-billing/config>` on the :guilabel:`Accounting Dashboard` or manually
set the :guilabel:`Journal` of the self-bill to the self-billing purchase journal.

.. _accounting/vendor_bills/self-billing/send:

Send a self-bill
----------------

To send a self-bill after :ref:`creating <accounting/vendor_bills/self-billing/create>` and
confirming it, click :guilabel:`Send`. In the :guilabel:`Send` popup window, select the sending
methods (i.e., :guilabel:`by Email`, :guilabel:`by Post`, and/or :guilabel:`by Peppol`), and click
:guilabel:`Send`.

.. note::
   - If sending self-bills via :ref:`Peppol <accounting/e-invoicing/peppol>`, first
     :ref:`configure your Peppol registration <accounting/e-invoicing/peppol-registration>`, then
     verify that the vendor is registered with Peppol. The :ref:`contact verification process
     <accounting/e-invoicing/contact-verification>` for vendors is the same as for customers except
     that vendors' contact records can be found via :menuselection:`Accounting --> Vendors -->

>      Vendors`.

Reviewer: I find the style (with "except that*) a bit weird for documentation? I'd rephrase:

```suggestion
     verify that the vendor is registered with Peppol. To do so, go to :menuselection:`Accounting --> Vendors --> Vendors`, then go through the :ref:`contact verification process <accounting/e-invoicing/contact-verification>` .
```
or something similar but direct

   - Not every Peppol access point supports self-billing. Check with your vendors to ensure that

>      their Peppol access point supports self-billing.

Reviewer: Could we maybe rephrase the second sentence to avoid the repetitions? 

```suggestion
   - Not every Peppol access point supports self-billing. Check with your suppliers to ensure theirs does.
```
or if you find a better solution, feel free to use it!
